README.rst

```diff
@@ -963,6 +982,79 @@ An example usage is shown below.
 **Note**: It will raise ``ImproperlyConfigured`` exception if the concerned chart
 configuration is not registered.
 
+Exceptions
+----------
+
+``TimeseriesWriteException``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+**Path**: ``openwisp_monitoring.db.exceptions.TimeseriesWriteException``
+
+If there is any failure due while writing data in timeseries database, this exception shall
+be raised with a helpful error message explaining the cause of the failure.
+This exception will normally be caught and the failed write task will be retried in the background
+so that there is no loss of data if failures occur due to overload of Timeseries server.
+You can read more about this retry mechanism at `OPENWISP_MONITORING_WRITE_RETRY_OPTIONS <#openwisp-monitoring-write-retry-options>`_.
+
+``InvalidMetricConfigException``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+**Path**: ``openwisp_monitoring.monitoring.exceptions.InvalidMetricConfigException``
+
+This exception shall be raised if the metric configuration is broken.
+
+``InvalidChartConfigException``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+**Path**: ``openwisp_monitoring.monitoring.exceptions.InvalidChartConfigException``
+
+This exception shall be raised if the chart configuration is broken.
+
+Rest API
+--------
+
+Live documentation
+~~~~~~~~~~~~~~~~~~
+
+.. image:: docs/api-doc.png
+
+A general live API documentation (following the OpenAPI specification) at ``/api/v1/docs/``.
+
+Browsable web interface
+~~~~~~~~~~~~~~~~~~~~~~~
+
+.. image:: docs/api-ui-1.png
+.. image:: docs/api-ui-2.png
+
+Additionally, opening any of the endpoints `listed below <#list-of-endpoints>`_
+directly in the browser will show the `browsable API interface of Django-REST-Framework
+<https://www.django-rest-framework.org/topics/browsable-api/>`_,
+which makes it even easier to find out the details of each endpoint.
+
+List of endpoints
+~~~~~~~~~~~~~~~~~
+
+Since the detailed explanation is contained in the `Live documentation <#live-documentation>`_
+and in the `Browsable web page <#browsable-web-interface>`_ of each point,
+here we'll provide just a list of the available endpoints,
+for further information please open the URL of the endpoint in your browser.
+
+Read Device and related Charts data
+###################################
+
+.. code-block:: text
+
+    GET /v1/monitoring/device/{pk}/?status=true
```

Comment:
isn't the key parameter necessary to access this?

Reply:
I would also specify calling without `?status=true` will only return the chart data.